Sources/MapboxNavigation/NavigationOptions.swift

```diff
@@ -53,14 +61,16 @@ open class NavigationOptions: NavigationCustomizable {
      - parameter voiceController: The voice controller that vocalizes spoken instructions along the route at the appropriate times.
      - parameter topBanner: The container view controller that presents the top banner.
      - parameter bottomBanner: The container view controller that presents the bottom banner.
+     - parameter predictiveCacheLocationOptions: Predictive Caching setup.
```

Comment:
Mention the `nil` behavior here too; this is where developers are more likely to encounter the option.